controls/V-92975.rb

```diff
@@ -1,11 +1,11 @@
 # encoding: UTF-8
 
 control 'V-92975' do
-  title "Windows Server 2019 must automatically remove or disable temporary user accounts after #{input('temporary_account_period_phrase')}."
+  title "Windows Server 2019 must automatically remove or disable temporary user accounts after #{input('temporary_account_period')*24} hours."
```

Comment:
Why do we store this as days instead of hours as an input if the text talks about hours?

Reply:
The original STIG seems like it uses hours, so we should probably use that here too.  Line 95 (and probably other locations) seems to do the comparison assuming days, but I think that's an easy enough change to make to how we're doing the assessment.

Reply:
The situation is analogous in 93509, where we're taking in seconds and converting them into minutes within the profile metadata.  There we should make the input take in minutes as well (and have the expected units set within the description), and then convert to seconds right before we do the test.

Reply:
This would be a simple change (to use the most common unit), however, since the value was originally set to a particular unit, the users will have their input files with values accordingly. Using the same input variable with different expected values would lead to some confusion; so historically we have been working around it. However, I don't mind changing the unit if that's the majority vote @em-c-rod @Amndeep7

Reply:
My opinion is that we release before changing just to have an option for anyone who wants a previous version.

Reply:
[Release created](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/releases/tag/1.3.0) and default input values changed https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/pull/107/commits/cf519f9ad158b115876f83f43feadb3f6189a7a6